eduNEXT/xblock-limesurvey
# refactor: load survey with access code from student view

### Description
This PR load the survey with the access code from student view. The action button was deleted, and therefore, the handler.

### How to test
1. Configure a survey as in [this PR](https://github.com/eduNEXT/xblock-limesurvey/pull/3).
2. In Studio add or edit the unit with the XBlock, and edit the configuration changing the Access key for API authentication and Survey ID.
3. From the LMS access to the unit with the XBlock.
4. You should see a view like this.
     ![image](https://github.com/eduNEXT/xblock-limesurvey/assets/64033729/1391acbd-d35e-4c9e-a269-b66d34f4112e)
